### PR TITLE
host: handle network drive for Windows filename

### PR DESCRIPTION
**Pull request by janlazo**
Fix https://github.com/neovim/node-client/issues/47#issuecomment-353062143

### CODE REVIEW

**Comment by janlazo** [comment]
Travis failure seems unrelated to the patch (use `let` to modify a value if running on Windows).
`stable` is an alias to `node` so nvm installs 9.3.0, latest stable version of node.
It should test on LTS versions (6.x.x, 8.x.x) instead.

**Comment by janlazo** [comment]
@billyvg Can you review my changes? I don't encounter any issues on node 6.12.x with my changes.

File: src/host/index.ts [hunk 1]
```diff
@@ -36,7 +36,11 @@ export class Host {
       logger.debug('host.handlePlugin: ', method);
 
       // Parse method name
-      const procInfo = method.split(':');
+      let procInfo = method.split(':');
+      if (process.platform === 'win32') {
+        const networkDrive = procInfo.shift();
```
Comment (billyvg) [inline src/host/index.ts]: Can you add a comment with example `procInfo` on windows?

**Comment by janlazo** [comment]
The integration tests should pass with #50 now.

**Comment by rhysd** [comment]
I'm merging this because CI failure is not related to changes of this PR. Thank you @janlazo

**Pull request merged**